policy/modules/system/getty.te

```diff
@@ -92,6 +96,12 @@ logging_send_syslog_msg(getty_t)
 
 miscfiles_read_localization(getty_t)
 
+allow getty_t sysctl_t:dir search;
+allow getty_t sysctl_kernel_t:dir search;
+allow getty_t sysctl_kernel_t:file { read open };
+
+allow getty_t init_t:unix_stream_socket connectto;
```

Comment:
Not allowed to reference init_t here. probably io.systemd.dynamicusers. which should be addressed with auth_use_nsswitch() when https://github.com/SELinuxProject/refpolicy/pull/269 is merged